week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -24,22 +24,22 @@ function greaterThan10() {
 
 function sortArray() {
   let letters = ["a", "n", "c", "e", "z", "f"];
-  let sortedLetters;
-
+  letters.sort();
+  letters.toString();
+  let sortedLetters = letters;
   return sortedLetters;
 }
 
 function first5() {
   let numbers = [1, 2, 3, 4, 5, 6, 7, 8];
-  let sliced;
-
+  numbers.length = 5;
+  let sliced = numbers;
   return sliced;
 }
 
 function get3rdIndex(arr) {
-  let index = 3;
-  let element;
-
+  let index = arr[3];
+  let element = index;
```

Comment:
Almost perfect! Try to think about what the variables are trying to express. The `index` is generally used to index the array. In you implementation you now made `index` the same as the `element`. Could you think of a way to change these 2 lines so that the variables keep their meaning?

Reply:
I will set the variable index equals with the arr and the element will be equals with index[3]